Sources/Parley/Views/ParleyView.swift

```diff
@@ -452,25 +452,25 @@ extension ParleyView: ParleyDelegate {
 
     func didChangePushEnabled(_ pushEnabled: Bool) {
         DispatchQueue.main.async { [weak self] in
-            guard let pollingService = self?.pollingService else { return }
-            if self?.offlineNotificationView.isHidden == false { return }
-            pushEnabled ? pollingService.stopRefreshing() : pollingService.startRefreshing()
+            guard let self else { return }
+            if self.offlineNotificationView.isHidden == false { return }
+            pushEnabled ? self.pollingService.stopRefreshing() : self.pollingService.startRefreshing()
             
-            self?.pushDisabledNotificationView.isHidden = pushEnabled
+            self.pushDisabledNotificationView.hide(pushEnabled)
```

Comment:
Just a FYI: After you've done a `guard let self` you can actually omit the `self` entirely :)